services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/BoosterNotificationStructureProvider.java

```diff
@@ -25,31 +19,28 @@
 @Component
 public class BoosterNotificationStructureProvider {
 
-  private static final Logger logger = LoggerFactory.getLogger(BoosterNotificationStructureProvider.class);
-
   public static final String EXPORT_BINARY_FILENAME = "export.bin";
 
   private final DistributionServiceConfig distributionServiceConfig;
-  private final CryptoProvider cryptoProvider;
-  private final DigitalGreenCertificateToCborMapping dgcToCborMapping;
   private final DigitalCovidCertificateClient digitalCovidCertificateClient;
+  private final BusinessRulesArchiveBuilder businessRulesArchiveBuilder;
 
   /**
    * Create an instance.
    */
   public BoosterNotificationStructureProvider(DistributionServiceConfig distributionServiceConfig,
       CryptoProvider cryptoProvider, DigitalGreenCertificateToCborMapping dgcToCborMapping,
```

Comment:
I think `CryptoProvider` and `DigitalGreenCertificateToCborMapping ` are not used anymore, right?